src/command_modules/vsts-cli-release/vsts/cli/release/_format.py

```diff
@@ -26,8 +26,11 @@ def _transform_release_row(row):
     table_row = OrderedDict()
     table_row['ID'] = row['id']
     table_row['Name'] = row['name']
+    table_row['DefinitionName'] = row['releaseDefinition']['name']
     table_row['CreatedBy'] = row['createdBy']['displayName']
-    table_row['Created on'] = row['createdOn']
+    created_on = dateutil.parser.parse(row['createdOn']).astimezone(dateutil.tz.tzlocal())
+    table_row['CreatedOn'] = str(created_on.date()) + ' ' + str(created_on.time())
```

Comment:
Multiple words are always space separated and Camel case in all other commands. Can you please make these consistent. (DefinitionName, CreatedOn, CreatedBy) Check build command table format for reference.

Reply:
@atbagga updated it.